bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -484,12 +485,16 @@ createCoinAccounts v meta = traverse (go <*> createCoinAccount v meta) names
 names :: NonEmpty String
 names = NEL.map safeCapitalize . NEL.fromList $ Prelude.take 2 $ words "mary elizabeth patricia jennifer linda barbara margaret susan dorothy jessica james john robert michael william david richard joseph charles thomas"
 
-formatB16PubKey :: Ed25519KeyPair -> Text
-formatB16PubKey = toB16Text . getPublic
+formatB16PubKey :: DynKeyPair -> Text
+formatB16PubKey = \case
+  DynEd25519KeyPair kp -> toB16Text $ getPublic kp
+  DynWebAuthnKeyPair pub _ _ -> toB16Text $ exportWebAuthnPublicKey pub
 
 safeCapitalize :: String -> String
 safeCapitalize = maybe [] (uncurry (:) . bimap toUpper (Prelude.map toLower)) . Data.List.uncons
 
+
+-- TODO: Use the new `assertCommand` function.
```

Comment:
Will this TODO be addressed here? If not, let's make a ticket.

Reply:
https://app.asana.com/0/1204132908184958/1206014030451021/f